### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by wismill**
Prepare release of:
- `unicode-data-0.3.1`
- `unicode-data-scripts-0.1.0`
- `unicode-data-security-0.1.0`

### CODE REVIEW

File: unicode-data/unicode-data.cabal [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cabal-version:       2.2
 name:                unicode-data
-version:             0.3.0
+version:             0.3.1
```
Comment (harendra-kumar) [inline unicode-data/unicode-data.cabal:3]: The commit message seems to have a typo. Should be "Bump" instead of "Dump"?

**Comment by harendra-kumar** [comment]
Looks good. Thanks for all the hard work you have put into this. It is good quality work.

**Review by harendra-kumar — APPROVED** [review]

File: unicode-data/unicode-data.cabal (hunk 1 above)
Comment (wismill) [reply unicode-data/unicode-data.cabal:3]: Good catch! I am going to squash the commit and merge.

**Comment by wismill** [comment]
@harendra-kumar candidates on Hackage:

- [`unicode-data-0.3.1`](https://hackage.haskell.org/package/unicode-data-0.3.1/candidate)
- [`unicode-data-scripts-0.1.0`](https://hackage.haskell.org/package/unicode-data-scripts-0.1.0/candidate)
- [`unicode-data-security-0.1.0`](https://hackage.haskell.org/package/unicode-data-security-0.1.0/candidate)

I will publish them soon this week if there is no negative feedback.

**Comment by harendra-kumar** [comment]
Most internal modules have haddock disabled but some don't.

**Comment by wismill** [comment]
> 1. Most internal modules have haddock disabled but some don't.

I am not sure what to do: as they are exposed we should also propose the docs, don't we?
>     2. A full stop is missing in this para: `The Haskell data structures are generated programmatically from the UCD The latest Unicode version supported by this library is [14.0.0](https://www.unicode.org/versions/Unicode14.0.0/).`

Ok will fix

>     3. APIs like `caseFoldMapping` use the `Unfold` type, but the `Unfold` type is not exposed, it is Internal. I think we can push these APIs to internal module and maybe expose them in future based on need. If we expose these we may have to expose the Unfold type as well.

No, it is exposed in `Unicode.Char.Case`.

**Comment by harendra-kumar** [comment]
> I am not sure what to do: as they are exposed we should also propose the docs, don't we?

I think we should. I thought haddock was intentionally disabled for all internal modules, so this looked inconsistent. But it seems it is only disabled for generated module. So it is fine.

I edited my suggestions above to add a few more.

**Pull request merged**